Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1324,9 +1325,17 @@ void CFrame::ParseHotkeys()
 
 	if (!Core::IsRunningAndStarted())
 	{
+		onceOnLaunch = true;
 		return;
 	}
 
+	if (onceOnLaunch)
+	{
+		onceOnLaunch = false;
+		if (SConfig::GetInstance().bAutoLoadSavestate)
+			State::LoadLastSaved();
```

Comment:
For me this option sounds like you want dolphin to start with this savestate (which is a very nice + needed feature), not to load it sometimes later. A "frame" may be very much :/

- Savestate not compatible: refuse to start
- no need to save another state to be able to rollback the load (much faster)
- no need to flush any internal state before